README.md

```diff
@@ -1 +1,31 @@
 # Website Redesign
+
+## How to use this repository
+
+1. Content creation step: First, you add or edit the actual content. Specifically, this means adding to or editing the various Markdown (filetype `.md`) files located in `content/`.
+   - TODO: We should gradually describe with more detail here.
+
+2. "Building" step: Second, you should "build" the Markdown content into actual HTML pages. This is done automatically when you run `python build.py` or `make` from the main directory. For installation of the python packages/environment, see below.
```

Comment:
Note for us to clarify that this is only for testing your work locally -> the build happens automatically via Actions and the html pages are built and deployed automatically to the `gh-pages` branch